src/conn/conn_api.c

```diff
@@ -1623,6 +1624,7 @@ wiredtiger_open(const char *home, WT_EVENT_HANDLER *event_handler,
 	WT_ERR(__wt_config_gets(session, cfg, "mmap", &cval));
 	conn->mmap = cval.val == 0 ? 0 : 1;
 
+	WT_ERR(__wt_sweep_config(session, cfg));
```

Comment:
These calls were alphabetized because their order didn't matter.  If there is a reason this call needs to be relocated now, we should comment that it must come before <other call>.